mobilecoind-dev-faucet/src/bin/main.rs

```diff
@@ -4,6 +4,7 @@
 
 #![deny(missing_docs)]
 #![feature(proc_macro_hygiene, decl_macro)]
+#![allow(clippy::let_unit_value)]
```

Comment:
What was triggering this lint? Is there a reason we're allowing it instead of changing the code to no longer trigger it?

Reply:
I couldn't figure out what caused this lint. The lint is normally triggered when you do something like `let value = ()`, but that's not going on here. I figured it had something to do with the internals of `rocket` and decided to just turn it off.